Display first line of task/setting of multiline description

ninjalama opened the pull request:
Fixes issue: #2218

ninjalama committed:
Only use first line for multiline descriptions

lightbend-cla-validator commented:
Hi @ninjalama,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

ninjalama committed:
Show all lines when running help <task>, but only first line when listing tasks

eed3si9n reviewed (approved):
LGTM
Thanks @ninjalama

dwijnand reviewed (approved):
LGTM! Thank you @ninjalama!
